docs/source/conf.py

```diff
@@ -42,8 +48,47 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
     'sphinx_mdinclude',
+    'doc_redirect',
+]
+
+md_parse_relative_links = True
+
+# references to RST files in 'docs' dir redirect to corresponding HTML
+# references to RST files in repo root (README/CHANGES) redirect to their equivalent HTML in 'docs' dir
+DOC_REDIRECT_DIRS = [
+    os.path.join(DOC_SRC_ROOT, "birdhouse"),
+]
+
+# files that will be copied over from root repo or linked birdhouse directory
+# under build directory (not copied over with same nested dir structure, directly under '_build/html')
```

Comment:
is there a way to keep the nested dir structure?  I just found a link that do not work, probably need nested dir structure:
https://birdhouse-deploy--261.org.readthedocs.build/en/261/optional-components/emu/docker-compose-extra.yml

Reply:
Will look into it. It will help keeping track of which file comes from it that can work. 

The hard case is that some references resolve automatically with the right nesting via symlink `docs/source/birdhouse -> birdhouse`, but others don't (ie: anything at the root above `birdhouse`), and there are "conflicting cases" like `CHANGES.md` that should not be auto-redirected since it is indirectly integrated via `docs/source/changes.rst`.